lib/tasks/manage.rake

```diff
@@ -1,6 +1,33 @@
 require "csv"
 
 namespace :manage do
+  desc "View all subscriptions for a subscriber"
+  task :view_subscriptions, [:email_address] => :environment do |_t, args|
+    email_address = args[:email_address]
+    subscriber = Subscriber.find_by_address(email_address)
+    abort("Cannot find any subscriber with email address #{email_address}.") if subscriber.nil?
+
+    results = subscriber.subscriptions.map do |subscription|
+      subscriber_list = SubscriberList.find(subscription.subscriber_list_id)
+      {
+        status: subscription.ended_at.present? ? "Inactive (#{subscription.ended_reason})" : "Active",
+        subscriber_list: "#{subscriber_list.slug} (id: #{subscriber_list.id})",
+        frequency: subscription.frequency.to_s,
+        timeline: "Subscribed #{subscription.created_at}#{subscription.ended_at.present? ? ", Ended #{subscription.ended_at}" : ''}",
+      }
+    end
+    columns = results.first.each_with_object({}) do |(col, _), h|
+      heading = col.to_s.humanize
```

Comment:
Could you explain why we're doing this below, my brain is struggling to parse this bit of code? 😄

Reply:
Converts the key (e.g. `subscriber_list`) to a heading we can display (e.g. `SubscriberList`).
The StackOverflow I took inspiration from had a hardcoded mapping instead (e.g. `subscriber_list: "SubscriberList"` but this added bloat and felt unnecessary.